src/atoms/second_order_cone/norm2.jl

```diff
@@ -1,59 +1,32 @@
-#############################################################################
-# LinearAlgebra.norm2.jl
-# Handles the euclidean norm (also called frobenius norm for matrices)
-# All expressions and atoms are subtpyes of AbstractExpr.
-# Please read expressions.jl first.
-#############################################################################
-
 mutable struct EucNormAtom <: AbstractExpr
     children::Tuple{AbstractExpr}
     size::Tuple{Int,Int}
 
-    function EucNormAtom(x::AbstractExpr)
-        children = (x,)
-        return new(children, (1, 1))
-    end
+    EucNormAtom(x::AbstractExpr) = new((x,), (1, 1))
 end
 
 head(io::IO, ::EucNormAtom) = print(io, "norm2")
 
-function Base.sign(x::EucNormAtom)
-    return Positive()
-end
+Base.sign(::EucNormAtom) = Positive()
 
-function monotonicity(x::EucNormAtom)
-    return (sign(x.children[1]) * Nondecreasing(),)
-end
+monotonicity(x::EucNormAtom) = (sign(x.children[1]) * Nondecreasing(),)
 
-function curvature(x::EucNormAtom)
-    return ConvexVexity()
-end
+curvature(::EucNormAtom) = ConvexVexity()
 
-function evaluate(x::EucNormAtom)
-    return norm(vec(evaluate(x.children[1])))
-end
+evaluate(x::EucNormAtom) = norm(vec(evaluate(x.children[1])))
 
-## Create a new variable euc_norm to represent the norm
```

Comment:
Maybe move it to a docstring ?

Reply:
This comment was outdated so I deleted it. The function did not in fact create a new variable `euc_norm` to represent the norm. I think it mean `t`, and okay, by reading the function I see it is building `[t, x] in SOC`. I don't think it needs a comment saying that.